<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [Mapnik-users] Strange installation problem
   </TITLE>
   <LINK REL="Index" HREF="http://lists.berlios.de/pipermail/mapnik-users/2008-February/index.html" >
   <LINK REL="made" HREF="mailto:mapnik-users%40lists.berlios.de?Subject=Re%3A%20%5BMapnik-users%5D%20Strange%20installation%20problem&In-Reply-To=%3C47C491A9.3030300%40neogeo-online.net%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000656.html">
   <LINK REL="Next"  HREF="000677.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[Mapnik-users] Strange installation problem</H1>
    <B>Guillaume Sueur</B> 
    <A HREF="mailto:mapnik-users%40lists.berlios.de?Subject=Re%3A%20%5BMapnik-users%5D%20Strange%20installation%20problem&In-Reply-To=%3C47C491A9.3030300%40neogeo-online.net%3E"
       TITLE="[Mapnik-users] Strange installation problem">guillaume.sueur at neogeo-online.net
       </A><BR>
    <I>Tue Feb 26 23:24:41 CET 2008</I>
    <P><UL>
        <LI>Previous message: <A HREF="000656.html">[Mapnik-users] Hel creating Tile files from shape files.
</A></li>
        <LI>Next message: <A HREF="000677.html">[Mapnik-users] Strange installation problem
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#676">[ date ]</a>
              <a href="thread.html#676">[ thread ]</a>
              <a href="subject.html#676">[ subject ]</a>
              <a href="author.html#676">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Hi list,

I've been compiling, installing and using mapnik on an ubuntu box with 
no problem.
I've tried to do the same on a second box, running Ubuntu-server
Both compiling and installing went fine
rundemo.py runs fine too
from mapnik import * is fine within python interpreter

BUT
running another python file (which works ok on the other box, it's just 
&quot;from mapnik import *&quot; and &quot;m = Map(600,600)&quot; ...) sends :
NameError: name 'Map' is not defined

if after that failure I run rundemo.py again, I've got :
The mapnik library and python bindings must have been compiled and 
installed successfully before running this script.

After a reboot of the server, it's fine again in python interpreter.

Am I forgetting something ? What's up ?

Thanks for your help !

-- 
Guillaume SUEUR


</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000656.html">[Mapnik-users] Hel creating Tile files from shape files.
</A></li>
	<LI>Next message: <A HREF="000677.html">[Mapnik-users] Strange installation problem
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#676">[ date ]</a>
              <a href="thread.html#676">[ thread ]</a>
              <a href="subject.html#676">[ subject ]</a>
              <a href="author.html#676">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.berlios.de/mailman/listinfo/mapnik-users">More information about the Mapnik-users
mailing list</a><br>
</body></html>
